### PR TITLE
Docker images clean up

### PR DESCRIPTION
**Pull request by salvatore-coppola**
Docker images have bean fixed. 
Base Alpine images has been updated.
ClokService now is disabled by default.
A new profile. ubuntu-20-nn has been added and it is the base package for docker images.

### CODE REVIEW

File: kura/container/README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ The JMX port defined is 9010 and the Remote debug port is 9011. Both ports are n
 
 ## Re-Building
```
Comment (mattdibi) [inline kura/container/README.md:54]: Can you update the docs to show the correct endpoint for the GUI? Right now it points to `localhost:8080` but is should be `localhost:8443` if the container is run using the command suggested in the docs.

I'm referring to the line 9 of the README:

> Once the image is started you can navigate your browser to http://localhost:8080 and log in using the credentials `admin` : `admin`.

**Comment by MMaiero** [comment]
Build is failing but due to tests not related to this PR.

File: kura/container/README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Use the following command to run it:
 
     docker run -d -p 8443:443 -t eclipse/kura
 
-Once the image is started you can navigate your browser to http://localhost:8080 and log in using the credentials `admin` : `admin`.
+Once the image is started you can navigate your browser to http://localhost:8443 and log in using the credentials `admin` : `admin`.
```
Comment (mattdibi) [inline kura/container/README.md]:
```suggestion
Once the image is started you can navigate your browser to https://localhost:8443 and log in using the credentials `admin` : `admin`.
```

**Review by mattdibi — COMMENTED** [review]
## Smoke tests

- [X] kura_alpine build
- [X] kura_ubi8 build
- [X] kura_alpine run (can correctly login to GUI)
- [X] kura_ubi8 run (can correctly login to GUI)

**Review by mattdibi — APPROVED** [review]
LGTM

**Pull request merged**